Examples/test-suite/dynamic_cast.i

```diff
@@ -17,14 +17,14 @@ public:
 };
 %}
 
-#if defined(SWIGJAVA) || defined(SWIGCSHARP) || defined(SWIGGO) || defined(SWIGD)
+#if defined(SWIGJAVA) || defined(SWIGCSHARP) || defined(SWIGGO) || defined(SWIGD)  || defined(SWIGC)
 %typemap(out) Foo *blah {
     Bar *downcast = dynamic_cast<Bar *>($1);
     *(Bar **)&$result = downcast;
 }
 #endif
 
-#if defined(SWIGJAVA)
+#if defined(SWIGJAVA) 
```

Comment:
Random trailing whitespace added.

Reply:
Added in the same a2dc2756c8, I'll revert it too.